apps/api/src/app.controller.ts

```diff
@@ -3,15 +3,23 @@ import { ClientProxy } from '@nestjs/microservices';
 
 @Controller()
 export class AppController {
-  constructor(@Inject('AUTH_SERVICE') private authService: ClientProxy) {}
+  constructor(
+    @Inject('AUTH_SERVICE') private authClientProxy: ClientProxy,
+    @Inject('PRESENCE_SERVICE') private presenceClientProxy: ClientProxy,
+  ) {}
 
   @Get('auth')
   async getUsers() {
-    return this.authService.send({ cmd: 'get-users' }, {});
+    return this.authClientProxy.send({ cmd: 'get-users' }, {});
   }
 
   @Post('auth')
   async postUser() {
-    return this.authService.send({ cmd: 'post-user' }, {});
+    return this.authClientProxy.send({ cmd: 'post-user' }, {});
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Consider more specific microservice command names</b></div>
<div id="fix">

Consider using a more descriptive command name for microservice communication. The current `cmd` values like 'post-user' and 'get-presence' might be too generic. Maybe consider using a more specific command name that includes the service name or action, such as 'auth:create-user' or 'presence:get-status'.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -18,2 +18,2 @@
 -    return this.authClientProxy.send({ cmd: 'post-user' }, {});
 +    return this.authClientProxy.send({ cmd: 'auth:create-user' }, {});
 @@ -23,2 +23,2 @@
 -    return this.presenceClientProxy.send({ cmd: 'get-presence' }, {});
 +    return this.presenceClientProxy.send({ cmd: 'presence:get-status' }, {});
```
</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/omkardongre/messenger-api/pull/6#issuecomment-2451737561>#148eb1</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged